tools/update-client

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# This script creates a new release of the Hypothesis browser extension
+# using the latest version of the client available on npm.
+
+set -eu
+
+LATEST_CLIENT_VERSION=$(npm view hypothesis version)
+
+# Update checkout
+git checkout master
+git fetch
+git reset --hard origin/master
```

Comment:
Thanks @fatbusinessman - I didn't know about the `--quiet` and `--exit-code` args to `git diff`.